CHANGELOG.md

```diff
@@ -5,10 +5,13 @@
   - Improvements
     - Verify parameters hash
     - Whitelist PrimeDAO Token
-    - Use subgraph v41_3
+    - Use subgraph v41_4 and arc.js 0.2.80
+    - Some UI/UX improvements
+    - Alchemy 2.0 banners are removed
 
   - Bugs Fixed
-    - Execute Calls button is displayed in Generic Scheme before it passed
+    - Execute Calls button is displayed in Generic Scheme proposal before it passed
```

Comment:
the multicall propose issue?

Reply:
No, it's the issue that I fixed in the patch. Will add the multicall propose issue as well.